src/useTimer.test.tsx

```diff
@@ -495,7 +495,7 @@ describe('State and callbacks', () => {
   it('should display "Running" text when timer is running', () => {
```

Comment:
We could update test description:
```suggestion
  it('should display "RUNNING" text when timer is running', () => {
```